packages/marshaller/src/ddl2/dashboard.ts

```diff
@@ -199,7 +207,22 @@ class DashboardDockPanel extends DockPanel implements IClosable, IVizPopupPanelO
 }
 
 export class Dashboard extends ChartPanel {
-    private _dockPanel: DashboardDockPanel;
+    _dockPanel: IDockPanel;
+
+    syncWidgets() {
+        console.log("syncWidgets currently does nothing");
+    }
+
+    private _designModeButton = new Button().faChar("fa-object-group").tooltip("Design Mode...")
+        .on("click", () => {
+            console.log("this", this);
+            if (this._dockPanel && (this._dockPanel as any).designMode) {
+                (this._dockPanel as any)
+                    .designMode(!(this._dockPanel as any).designMode())
+                    .render()
+                    ;
+            }
+        });
```

Comment:
Would be better if we didn't need a "designMode" button - maybe the Grid layout drag zone is limited to the titlebar?

Reply:
It's common for some widgets to hide their titlebar. Could we live with this designMode button for now and then the `AbsoluteGrid` will make use of that best-of-both-worlds solution?